scripts/docker-entrypoint.sh

```diff
@@ -60,6 +60,7 @@ initialize() {
     map_uidgid
     set_permissions
     migrations
+    sudo -HEu paperless "/usr/src/paperless/src/manage.py" "collectstatic" "--clear" "--no-input"
```

Comment:
Nit: Even though it is only one line, I'd feel it be better placed in its own function, similar to `migrations`.